### PR TITLE
Add sequnece labeling predict

### PR DESCRIPTION
**Pull request by Steffy-zxf**
Add the file predict.py to the demo sequence-labeling.

### CODE REVIEW

File: demo/sequence-labeling/predict.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            seq_label_task.variable("seq_len").name
+        ]
+
+        # classification probability tensor
+        probs = seq_label_task.variable("probs")
+
+        # load best model checkpoint
+        fluid.io.load_persistables(exe, args.checkpoint_dir)
+
+        inference_program = program.clone(for_test=True)
```
Comment (Steffy-zxf) [inline demo/sequence-labeling/predict.py:91]: 有待优化。TBD

File: demo/sequence-labeling/predict.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            name="label", shape=[args.max_seq_len, 1], dtype='int64')
+        seq_len = fluid.layers.data(name="seq_len", shape=[1], dtype='int64')
+
+        # Use "sequence_outputs" for token-level output.
+        sequence_output = output_dict["sequence_output"]
+
+        # Setup feed list for data feeder
+        # Must feed all the tensor of ERNIE's module need
+        # Compared to classification task, we need add seq_len tensor to feedlist
+        feed_list = [
```
Comment (Steffy-zxf) [inline demo/sequence-labeling/predict.py]: predict是feed_list需要去掉label

File: demo/sequence-labeling/predict.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+            input_dict["input_ids"].name, input_dict["position_ids"].name,
+            input_dict["segment_ids"].name, input_dict["input_mask"].name,
+            label.name, seq_len
+        ]
```
Comment (Steffy-zxf) [inline demo/sequence-labeling/predict.py:70]: predict是feed_list需要去掉label

**Review by ZeyuChen — APPROVED** [review]
LGTM

**Pull request merged**